tools/objc/libtool.sh

```diff
@@ -35,7 +35,8 @@ fi
 WRAPPER="${MY_LOCATION}/xcrunwrapper.sh"
 
 if [ ! -f "${MY_LOCATION}"/libtool_check_unique ] ; then
-    echo "libtool_check_unique not found. Please file an issue at github.com/bazelbuild/bazel"
+  echo "libtool_check_unique not found. Please file an issue at github.com/bazelbuild/bazel"
+  exit 1
```

Comment:
now that this script has exited for a while I think erroring if it doesn't exist is reasonable